omnipaxos_core/tests/reconnect_test.rs

```diff
@@ -0,0 +1,260 @@
+pub mod utils;
+
+use kompact::prelude::{promise, Ask, FutureCollection};
+use omnipaxos_core::{
+    ballot_leader_election::Ballot,
+    messages::{sequence_paxos::PaxosMsg, Message},
+    storage::Snapshot,
+    util::LogEntry,
+};
+use serial_test::serial;
+use std::{thread, time::Duration};
+use utils::{LatestValue, TestConfig, TestSystem, Value};
+
+const SLEEP_TIMEOUT: Duration = Duration::from_secs(1);
+
+/// Verifies that a leader sends out AcceptSync messages
+/// with increasing sequence numbers.
+#[test]
+#[serial]
+fn ascending_accept_sequence_test() {
+    // Start Kompact system
+    let cfg = TestConfig::load("reconnect_test").expect("Test config couldn't be loaded");
+    let mut sys = TestSystem::with(
+        cfg.num_nodes,
+        cfg.election_timeout,
+        cfg.num_threads,
+        cfg.storage_type,
+    );
+    sys.start_all_nodes();
+
+    // Propose some values so that a leader is elected
+    make_proposals(&sys, &cfg, 1, 1, 5);
```

Comment:
Make all these kind of test variables into a variable/constant defined in the beginning of the test so it's easy to read the code and get an idea of what will happen in the test